jre/javatests/com/google/j2cl/jre/java9/util/OptionalTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2023 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.j2cl.jre.java9.util;
+
+import com.google.j2cl.jre.java.util.EmulTestBase;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+/**
+ * Tests for java.util.Optional Java 9 API emulation.
+ */
+public class OptionalTest extends EmulTestBase {
+  public void testIfPresentOrElse() {
```

Comment:
Some of these were already tested under OptionalTest. Could you remove the tests here and if a scenario is missing could you update the other one?

Reply:
I agree, this test is redundant